src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
                                             MAV_CMD_DO_SET_CAM_TRIGG_DIST,
                                             MAV_FRAME_MISSION,
-                                            0.0,                    // trigger distance (off)
+                                            -1.0,                   // trigger distance (off)
```

Comment:
No idea if -1 works with ArduPilot. I know and have tested 0 to work with ArduPilot.